src/lozad.js

```diff
@@ -55,6 +58,13 @@ export default function (selector = '.lozad', options = {}) {
         load(elements[i])
         markAsLoaded(elements[i])
       }
+    },
+    triggerLoad(selector) {
+      const elements = document.querySelectorAll(selector)
```

Comment:
A better solution would be use the [matches](https://developer.mozilla.org/en-US/docs/Web/API/Element/matches) method. Because the user will only provide a selector that will differentiate that image from the others. What do you think?

Reply:
Maybe, but we should be careful about browser support (see [caniuse](http://caniuse.com/#feat=matchesselector), IE 11 and older Edge require a prefix). We could use feature detection, though, but only if it's worth it performance-wise.

Reply:
But it has a better support than the [intersection observer](http://caniuse.com/#feat=intersectionobserver), that it is the basis of the project.

Reply:
Yes, but if we polyfill `IntersectionObserver`, we're fine, and I don't think we want to polyfill one more thing :)

Reply:
Hmm, I suppose, we could simply do something like `matches | msMatches | webkitMatches`, that should be an easy way to improve browser support a lot.

Reply:
Makes senses. I just updated with that.